librabft-v2/src/node.rs

```diff
@@ -71,8 +71,7 @@ impl CommitTracker {
 }
 
 /// Initial configuration of LibraBFTv2 node.
-#[derive(PartialEq, Clone, Debug, Serialize, Deserialize)]
-#[cfg_attr(test, derive(Default))]
```

Comment:
The generated default `NodeConfig` is really meaningless and only suitable for tests. I feel that we should either write a more meaningful default (but the Duration unit is unknown?) or keep it private.